### DISCUSSION TITLE
Ender 3 V2 print volume size discrepancy

### DISCUSSION THREAD

**Discussion by RanceH** [Jyers/Marlin#1864] [Q&A] [answered]
Got the Ender 3 V2 for Christmas and was a little disappointed.  Not a single slicer on the market has direct support for it, including Creality's own windows only slicer. Crazy right? almost makes me wish I'd bought an original 3.

Anyway, updated the firmware to Jyers last night and was getting ready to level the bed manually when I discovered an issue.

The firmware (pre-compiled .bin file) reports that the print volume is 230x230x250.  Creality and every other Howto I could find for adding support for a 3V2 to a slicer manually says that the print volume is 220x220x250.  Just wondering if there is a problem, and where that problem could be, if it existed at all.

Thanks for reading.

**Comment by K2Kevin** [comment, marked as answer]
> 

That is strange, I also have an Ender 3v2 and every slicer supports it (If its not in the list, choose Ender 3, basically the same exact profiles compared to those that actually list 3v2).  Maybe you expect some magic default setting to give you perfect prints?  That is not going to happen with any 3d Printer. Every printer is different, all profiles are a starting point.  It takes a lot of profile adjustments to get the best out of most, if not all of these printers.  

Bed size and safe printing area varies.  The firmware will report the actual size while your slicer does NOT want to print on the edge, so that size should be reduced.

Im using **SuperSlicer** (its basically Prusa slicer with more settings), I have also used CURA, and IdeaMaker.  And I get some pretty clean prints on an Ender 3v2.  Heres my new dual fan duct print.

![IMG_1399](https://user-images.githubusercontent.com/27362912/155185687-c823e589-d124-4986-8df9-34050709f068.jpeg)

**Reply by RanceH** [reply]
Thanks for the time to reply.  I checked out SuperSlicer and the interface is so much cleaner than Cura.  I haven't really spent time mastering it, but found it awkward, and hard to understand.  SuperSlicer is really nice and I got further in 10ish minutes with it than I have with Cura.

I assumed that the explanation the size discrepancy was one of actual size vs usable size, but without knowing for sure, assumptions are mistakes that haven't happened yet.  I'm not expecting "magic" I'm expecting an ordeal of sorts.  But also want to be comfortable with the foundation that I'm building on.

**Reply by grrmisfit** [reply]
I will probably say this wrong as im new, but from my reading yes the bed printable area is 230x230 but they reduce it to 220x220 to accommodate the clips and hardware issues. https://all3dp.com/2/ender-3-bed-size-what-is-it-really/

**Reply by K2Kevin** [reply]
Pretty accurate.  I now use a PEI spring steal magnetic bed (magnet on backside of the OEM glass bed). I did gain some bed area that I don't use often.

**Reply by K2Kevin** [reply]
> Thanks for the time to reply.  I checked out SuperSlicer and the interface is so much cleaner than Cura.  I haven't really spent time mastering it, but found it awkward, and hard to understand.  SuperSlicer is really nice and I got further in 10ish minutes with it than I have with Cura.
> 
> 
> 
>
Completely agree!!!
